README.md

```diff
@@ -112,84 +125,141 @@ The content of your document is retrieved.
 *   [decodeDocument](#decodedocument)
     *   [Parameters](#parameters-2)
 *   [IEncryptionResults](#iencryptionresults)
-*   [encryptString](#encryptstring)
-    *   [Parameters](#parameters-3)
-*   [decryptString](#decryptstring)
-    *   [Parameters](#parameters-4)
+    *   [encryptString](#encryptstring)
```

Comment:
the original hierarchy is correct, but IEncryptionResults is an interface not a function which is why it has no parameters specified. Would it be clearer if we documented the interface properties? 

https://github.com/Open-Attestation/oa-encryption/blob/master/src/index.ts#L59-L65

Reply:
Updated the `IEncryptionResults` section and TOC. Moved it below `decryptString`.